build/getLastTag.mjs

```diff
@@ -2,7 +2,7 @@ import {getLastTag} from '@coveo/semantic-monorepo-tools';
 import url from 'url';
 
 const VERSION_PREFIX = 'v';
-const lastTag = getLastTag(VERSION_PREFIX)?.[0];
+const lastTag = await getLastTag(VERSION_PREFIX)?.[0];
```

Comment:
I pulled your branch and ran the script (`node build/publishNewVersion.mjs --dry`) and it fails on that line, it doesn't return an array anymore

```suggestion
const lastTag = await getLastTag(VERSION_PREFIX);
```